lib/rudder/analytics/transport.rb

```diff
@@ -29,7 +29,7 @@ def initialize(config)
         uri = URI(config.data_plane_url)
 
         http = Net::HTTP.new(uri.host, uri.port)
-        http.use_ssl = config.ssl || true
+        http.use_ssl = config.ssl.nil? ? true : config.ssl
```

Comment:
do not know ruby, is tis both nil and undefined?

Reply:
Yeah. Same.